drivers/lm75/include/lm75_params.h

```diff
@@ -146,7 +146,7 @@ extern "C" {
                                  .gpio_alarm      = LM75_PARAM_INT, \
                                  .conv_rate       = LM75A_CONV_RATE, \
                                  .i2c_bus         = LM75_PARAMS_I2C, \
-                                 .i2c_addr        = CONFIG_I2C_ADDR, \
+                                 .i2c_addr        = CONFIG_LM75_I2C_ADDR, \
                                  .shutdown_mode   = CONFIG_OPERATION_MODE, \
                                  .tm_mode         = CONFIG_THERMOSTAT_MODE, \
                                  .polarity        = CONFIG_OS_POLARITY, \
```

Comment:
eh now I noticed you also didn't prefix the other `CONFIG_` defines - want to do that real quick?